packages/extract-react-types/converters-typescript.test.js

```diff
@@ -690,3 +690,18 @@ cases(
   },
   TESTS
 );
+
+test('Typescript quoted property name is converted properly', () => {
```

Comment:
Would be great if we could also test against props written with `type`

```ts
type Props = {....} 
```

Reply:
Good idea, will do 🙂